<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INLTP-SHILPIM" ignored="0" name="Reports(failed)" tests="1" failures="1" timestamp="05 May 2020 08:26:42 GMT" time="85.54" errors="0">
  <testcase name="reviewProgressByRevReport" time="28.131" classname="testScriptsSmoke.TS_012_ValidateReportsFunctionality">
    <failure type="java.lang.AssertionError" message="expected [Error !] but found [Success !]">
      <![CDATA[java.lang.AssertionError: expected [Error !] but found [Success !]
	at pageFactory.ReviewProgressByReviewerReportPage.successMsgConfirmation(ReviewProgressByReviewerReportPage.java:195)
	at pageFactory.ReviewProgressByReviewerReportPage.ValidateReviewProgressReportShareandSchdule(ReviewProgressByReviewerReportPage.java:118)
	at testScriptsSmoke.TS_012_ValidateReportsFunctionality.reviewProgressByRevReport(TS_012_ValidateReportsFunctionality.java:117)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- reviewProgressByRevReport -->
</testsuite> <!-- Reports(failed) -->
